openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -150,6 +150,30 @@ def __init__(self, **data):
         if any(len(pur_list) > 2 for pur_list in self.data.values()):
             raise NotImplementedError("Can't stitch together results yet")
 
+    def _get_filenames(self, file_type) -> list[pathlib.Path]:
+        """
+        Get a list of paths to files
+
+        Parameters
+        ----------
+        file_type: str
+          Str of the dictionary entry for which the file paths should be
+          returned
+        Returns
+        -------
+        files : list[pathlib.Path]
+          list of paths (pathlib.Path) to the files
+        """
+        file_paths = [
+            pus[0].outputs[file_type]
+            for pus in self.data.values()
+            if "GromacsMDRunUnit" in pus[0].source_key and file_type in pus[0].outputs
+        ]
+        if len(file_paths) == 0:
+            file_paths = None
```

Comment:
```suggestion
        if not file_paths:
            return None
```
Very minor thing, just a slightly more pythonic way of doing this.
The `if not` comes from the pep8 programming recommendations which you can see here: https://peps.python.org/pep-0008/#programming-recommendations

"For sequences, (strings, lists, tuples), use the fact that empty sequences are false"